config/config.yaml

```diff
@@ -12,6 +12,8 @@ polygon: [[0.5, -0.5],[0.5, 0.5], [-0.5, 0.5], [-0.5, -0.5]] #in float
 
 polygon_topic_name: "/lidar_crop_polygon" #topic name of polygon
 
```

Comment:
can you please write a comment about angle min and angle max, about the units, and format you are taking in the parameter.
for example
Units: degrees 
angle_min: [angle_min_for_scan1, angle_min_for_scan2,...]